<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>550</width>
           <height>385</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblSketchImage">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>Sketch Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblSScaling">
          <property name="text">
           <string>Scaling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hsldSScaling">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="pageStep">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbSScaling">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSDock">
          <property name="text">
           <string>Dock</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hsldSScaling</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spbSScaling</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>752</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbSScaling</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsldSScaling</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>752</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
